configure.ac

```diff
@@ -189,6 +189,16 @@ ENABLE_PLUGIN_WITH_DEP(adplug,
     ADLIB,
     adplug)
 
+if test $have_adplug = yes ; then
```

Comment:
It would be nice to update the Meson side too.

Reply:
In progress... #79 is a component of this. It appears meson builds were broken on mingw32 and potentially other platforms.

Reply:
Fixed in efa1585.

Reply:
Fixed in 6258fd6.